packages/fetchai/skills/tac_control_contract/helpers.py

```diff
@@ -43,9 +43,10 @@ def generate_good_id_to_name(nb_goods: int, contract: Contract) -> Dict[str, str
     max_number_of_digits = math.ceil(math.log10(nb_goods))
     token_ids = contract.create_token_ids(TOKEN_TYPE, nb_goods)  # type: ignore
     string_format = "tac_good_{:0" + str(max_number_of_digits) + "}"
-    return {
+    token_ids_dict = {
```

Comment:
note, this is mapping token_names to token_ids. You should check if the map should not be the other way round as the name suggests (generate_good_id_to_name)

Reply:
should be
```
"""
token_ids = contract.create_token_ids(TOKEN_TYPE, nb_goods)
token_ids_dict = { token_id: "NFT_{}".format(token_id) for token_id in token_ids }
```

Reply:
and change signature to `-> Dict[int, str]`